lib/fog/openstack.rb

```diff
@@ -106,6 +110,7 @@ module OpenStack
     service(:monitoring,    'Monitoring')
     service(:workflow,      'Workflow')
     service(:dns,           'DNS')
+    service(:share,         'Share')
```

Comment:
I actually initially missed that, I think the name could be `:shared_file_system`.